### ISSUE TITLE
Chinese User Guide

### ISSUE THREAD

**Issue by robyle** [dotnetcore/mocha#14]
Build a Chinese User Guide Document for Chinese User Guide and Integrated into system

**Comment by liuhaoyang** [comment]
Of course, we will give priority to providing Chinese developer guidance documents.
